team.html

```diff
@@ -52,22 +52,30 @@
                 <ul class="navbar-nav ml-auto">
                     <li class="nav-item active">
                         <a class="nav-link font-weight-bold nav-items" href="index.html">Home </a>
+                        <hr class="hr">
```

Comment:
If there is no styles for class `hr` then we can remove it for all the tags.

Reply:
I have tried removing hr class but it gives a different style which is not required so i remained it as same as i have used it in style.css

Reply:
okay